bedrock/firefox/templates/firefox/australis/whatsnew.html

```diff
@@ -55,10 +55,10 @@
 {% block aside_content %}
   <section class="learn-more billboard">
     <ul>
-      <li><a href="https://support.mozilla.org/products/firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Need help?') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/learn-more-about-the-design-of-new-firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Learn about the new design') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/what-happened-to-the-add-on-bar" target="_blank" title="{{ _('Opens new window') }}">{{ _('Where did my add-on bar go?') }}</a></li>
-      <li><a href="https://support.mozilla.org/kb/common-questions-after-updating-to-new-firefox" target="_blank" title="{{ _('Opens new window') }}">{{ _('Frequently asked questions') }}</a></li>
+      <li><a href="https://support.mozilla.org/products/firefox">{{ _('Need help?') }}</a></li>
```

Comment:
It looks like these links are still opening in new tabs dues to some [JS happening here](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/australis/common.js#L79-L89). Please delete the `.on('click')` handler here so the code just reads:

```
// track learn more links
$('.learn-more a').each(function() {
    $(this).attr({
        'data-page-name': pageId,
        'data-interaction': 'button click',
        'data-element-location': 'Get Started with Sync'
    });
});
```